lib/docx/body-reader.js

```diff
@@ -300,7 +300,7 @@ function BodyReader(options) {
     
     function readTableRow(element) {
         var properties = element.firstOrEmpty("w:trPr");
-        var isHeader = !!properties.first("w:tblHeader");
+        var isHeader = !!(properties.first("w:tblHeader") || properties.children && properties.children[0].attributes['w:firstRow'] === '1');
```

Comment:
Plucking out whatever happens to be the first child seems dangerous to me. What is the first child in this case?

Also, if `w:firstRow` is a normal boolean property (I haven't had time to check the spec), then it should use the existing boolean parsing code to handle other values such as `true`.